README.md

```diff
@@ -77,5 +77,5 @@ This includes not just bugs and feature requests but also general questions abou
 
 ## Related packages
 
-- [probdiffeq](https://pnkraemer.github.io/probdiffeq/): Fast and feature-rich filtering-based probabilistic ODE solvers in JAX.
-- [ProbNum](https://probnum.readthedocs.io/en/latest/): Probabilistic numerics in Python. It has not only probabilistic ODE solvers, but also probabilistic linear solvers, Bayesian quadrature, and many filtering and smoothing implementations.
+- [ProbDiffEq](https://pnkraemer.github.io/probdiffeq/) is similar in scope to ProbNumDiffEq.jl and it provides fast and feature-rich probabilistic ODE solvers but is implemented in Python and built on JAX.
```

Comment:
https://github.com/nathanaelbosch/ProbNumDiffEq.jl/blob/main/docs/src/index.md should be updated accordingly

Reply:
Good catch, thanks a lot!